src/components/CommitChartLabel/index.tsx

```diff
@@ -12,25 +21,37 @@ interface Props {
   createTime: Date;
   author: string;
   message: string;
+  versionId: string;
 }
 
 const CommitChartLabel: React.FC<Props> = ({
   githash,
   createTime,
   author,
   message,
+  versionId,
 }) => {
   const createDate = new Date(createTime);
   const shortenMessage = message.length > MAX_CHAR;
   const shortenedMessage = message.substring(0, MAX_CHAR - 3).concat("...");
+  const { data: configData } = useQuery<
+    GetSpruceConfigQuery,
+    GetSpruceConfigQueryVariables
+  >(GET_SPRUCE_CONFIG);
+  const jiraHost = configData?.spruceConfig?.jira?.host;
 
   return (
     <LabelContainer data-cy="commit-label">
       <LabelText>
-        {githash.substr(0, 6)} {shortDate(createDate)}
+        <StyledRouterLink to={getVersionRoute(versionId)}>
+          {githash.substring(0, 6)}
```

Comment:
Based on a [quick search](https://github.com/evergreen-ci/spruce/search?q=%22substring%22) it looks like we're shortening git hashes somewhat inconsistently across Spruce. Is there a particular reason to use a 5-character hash here? If not, it may be nice to standardize (the [default is 7 characters](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection#_short_sha_1), fwiw). This may just make sense as a followup ticket!

Reply:
Good point. It seems like we show hash lengths are between 5-7 characters in various parts of our app. I will ask in the design sync whether we should just stick with one length.

Reply:
I will change this to 5 for now since that's what we currently have in the legacy ui labels.